crates/syntax/src/ast/edit.rs

```diff
@@ -93,6 +93,22 @@ where
     }
 }
 
+impl ast::Impl {
+    #[must_use]
+    pub fn with_items(&self, items: ast::AssocItemList) -> ast::Impl {
```

Comment:
Let's be maximally boring here: `with_assoc_item_list`. One day, we'll auto-generate this....